src/scancode/resource.py

```diff
@@ -153,6 +154,7 @@ def from_dict(cls, **kwargs):
             'notice',
             'start_timestamp',
             'end_timestamp',
+            'time_taken',
```

Comment:
Could you call this `duration` instead? that's more consistent with the way we name such filed otherwise